### PR TITLE
`avalanche key transfer`

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
Documents the CLI for a native P-chain address to P-chain address. Include notes about using this funcitonality for cross-primary-network-chain transfers (i.e. sending AVAX from C to P-chain).

### CODE REVIEW

File: content/docs/tooling/transactions/native-send.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Send AVAX on C/P-Chain
+description: Learn how to execute a native transfer on the C or P-Chain using the Avalanche CLI.
+---
+
+# Prerequisites
+
+- Install the [Avalanche CLI](https://docs.avax.network/tooling/guides/get-avalanche-cli).
+- Use the CLI to [create a key](https://docs.avax.network/tooling/avalanche-cli#key-create).
+- Fund the key with AVAX. You can use the [faucet](https://test.core.app/tools/testnet-faucet/?subnet=c&token=c) with coupon code `devrel-avax-0112` to get testnet AVAX.
```
Comment (owenwahlgren) [inline content/docs/tooling/transactions/native-send.mdx:10]: is it ok to give out this coupon? I thought Ash said something about this yesterday

File: content/docs/tooling/transactions/native-send.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+Review the transaction details and confirm/abort:
+
+```zsh
+this operation is going to:
+- send 100.000000000 AVAX from P-avax1gmuqt8xg9j4h88kj3hyprt23nf50azlfg8txn2 to destination address P-avax1f630gvct4ht35ragcheapnn2n5cv2tkmq73ec0
+- take a fee of 0.001000000 AVAX from source address P-avax1gmuqt8xg9j4h88kj3hyprt23nf50azlfg8txn2
+? Confirm transfer:
+    No
+  ▸ Yes
+```
```
Comment (owenwahlgren) [inline content/docs/tooling/transactions/native-send.mdx:109]: is there a way to check transfer for successful?

File: content/docs/tooling/transactions/native-send.mdx (hunk 1 above)
Comment (meaghanfitzgerald) [reply content/docs/tooling/transactions/native-send.mdx:10]: this is the general coupon, unrelated to Ash's convo. If this coupon is abused we can terminate it's use pretty easily. I wouldn't want anyone to follow this tutorial and be completely unable to get testnet tokens.

File: content/docs/tooling/transactions/native-send.mdx (hunk 2 above)
Comment (meaghanfitzgerald) [reply content/docs/tooling/transactions/native-send.mdx:109]: you could look up your sender address on any of the public block explorers. You can also check your key's balance with `avalanche key list` and selecting the network. I will add this in.

**Review by owenwahlgren — APPROVED** [review]
LGTM! 🚀

**Pull request merged**